### ISSUE TITLE
Home button doesn't work when plugin is active

### ISSUE THREAD

**Issue by tbsp** [mikewii/Luma3DS-plugin#23]
Is there a setting I need to adjust for this, or is it a limitation of Luma3DS plugins?

I'm aware I can reboot via the Rosalina menu, but I'd prefer to use the home button.

**Comment by yaboifuso** [comment]
Something may be wrong with either your Luma setup or your 3DS - I'm able to home button and switch apps with the plugin enabled on both my n3ds and my test o3ds with the v1.0.6.1 plugin.

**Comment by tbsp** [comment]
Should I be using the newest Luma3DS release boot.firm (10.2), or the boot.firm from the MH plugin releases (which is 10.0.1 or so)? Or am I misunderstanding the installation instructions for this plugin, which seem to indicating copying boot.firm to the SD root (/), which would overwrite the Luma3DS boot.firm.

When I disabled plugin loading the home button worked without issue, so the button itself is likely fine.

**Comment by yaboifuso** [comment]
Use the boot.firm provided with the plugin. It's still Luma, but its a modified version by Mikewii to include some of his own custom features. Overwrite your boot.firm on your root with it (or just rename your current boot.firm to boot.firm.bak to save it) and then place the default.3gx plugin in your luma/plugins folder. There should be no reason the home button doesn't work with this plugin,

**Comment by tbsp** [comment]
Yeah, I figured that's what the boot.firm was, and had done exactly what you described. I've never had issues with buttons not working before.

If I disable plugin loading, or delete the default.3gx file (so there's no plugin to load), the home button behaves as expected. Not sure what's causing it, but thanks for your help.
